plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -163,6 +182,10 @@ def main():
                 type='list',
                 elements='str',
             ),
+            apply_live=dict(
+                required=False,
+                type='bool',
```

Comment:
```suggestion
                type='bool',
                default='no',
```

Reply:
Please do not use `default='no'`, but `default=False`. `'no'` is a string, not a Python boolean.

Reply:
Thanks @felixfontein